packages/lodestar/src/constants/network.ts

```diff
@@ -16,6 +19,46 @@ export enum Method {
   BeaconBlocksByRoot = "beacon_blocks_by_root",
 }
 
+export enum MethodResponseType {
+  SingleRespone = "SingleRespone",
```

Comment:
Typo

Reply:
I already fixed that in #841 which is based on this branch so we can merge this